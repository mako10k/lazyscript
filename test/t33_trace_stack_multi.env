LAZYSCRIPT_TRACE_STACK_DEPTH=2
