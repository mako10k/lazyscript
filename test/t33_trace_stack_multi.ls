# ネストした呼び出しで未定義参照を発生させ、2フレーム分のスタックを期待
!{ ( \ ~x -> ~x ) ~undefined_symbol };
