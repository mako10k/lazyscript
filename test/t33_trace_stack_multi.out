E: /workspaces/lazyscript/test/t33_trace_stack_multi.ls:2.19-36: undefined reference: ~undefined_symbol
E: #err "undefined reference"
